python/lsst/ts/ATDome/mock_controller.py

```diff
@@ -108,19 +118,48 @@ class MockDomeController:
     * Encoder counts are not supported; reported values are bogus.
     * Door sequencing is not supported; doors move as if independent.
     """
-    def __init__(self, port, door_time=1, az_vel=6, home_az=10, home_az_overshoot=1, home_az_vel=1):
+
+    def __init__(
+        self,
+        port,
+        door_time=1,
+        az_vel=6,
+        home_az=10,
+        home_az_overshoot=1,
+        home_az_vel=1,
+    ):
         self.port = port
         self.door_time = door_time
         self.az_vel = Angle(az_vel, u.deg)
+        self.high_speed = Angle(5, u.deg)
+        self.coast = Angle(0.2, u.deg)
+        self.tolerance = Angle(1.0, u.deg)
         self.home_az = Angle(10, u.deg)
         self.home_az_overshoot = Angle(home_az_overshoot, u.deg)
         self.home_az_vel = Angle(home_az_vel, u.deg)
+        self.encoder_counts_per_360 = 4018143232
         self.az_actuator = AzActuator(start_position=0, speed=az_vel)
-        self.door_actuators = dict((enum, simactuators.PointToPointActuator(min_position=0,
-                                                                            max_position=100,
-                                                                            start_position=0,
-                                                                            speed=100/door_time))
-                                   for enum in Door)
+        self.az_move_timeout = 120
+        self.watchdog_reset_time = 600
+        self.dropout_timer = 100
+        self.reverse_delay = 2
+        self.main_door_encoder_closed = 1856
```

Comment:
where do these values from from?

Reply:
Maybe look at the video i sent you and update these values?

Reply:
These values came from a dump of full status. They seem very different, so I am wondering if the values in full status are different than those in the EUI. But the last full status dump I got had 0s for all of those values so I don't have any recent data to compare to. I'll just change them and home they are right. If they don't match the EUI then perhaps we should ask the vendor to fix the full status.

Reply:
![image](https://user-images.githubusercontent.com/14132079/74054246-e987be00-499a-11ea-974b-c475c39424ef.png)

Reply:
![image](https://user-images.githubusercontent.com/14132079/74054270-f60c1680-499a-11ea-8bfb-d4f3606c630a.png)

Reply:
Those are taken about 30s apart... but no changes were made.

Reply:
Thank you. That is very helpful. I am glad the values are the same. I switched to "long long" for all encoder outputs in ts_xml.